lib/alephant/logger/cloudwatch.rb

```diff
@@ -30,7 +30,7 @@ def preset_defaults
       def parse(dimensions)
         dimensions.map do |name, value|
           {
```

Comment:
This still won't fix it, the main issue is the keys of this hash are symbols not strings.